::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
: Copyright (c) 2017 Enea AB and others.
:
: All rights reserved. This program and the accompanying materials
: are made available under the terms of the Apache License, Version 2.0
: which accompanies this distribution, and is available at
: http://www.apache.org/licenses/LICENSE-2.0
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
From: Stanislaw Bogatkin <sbogatkin@mirantis.com>
Date: Mon, 13 Mar 2017 14:13:34 +0300
Subject: [PATCH] Increase reboot task timeout

To avoid broken provision on baremetal servers, increase reboot
task timeout to 15 minutes.

Change-Id: Ia7b9aa96407dae5819420146f5e1e934031b662c
Closes-Bug: #1671793
---
 graphs/provision/tasks.yaml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/graphs/provision/tasks.yaml b/graphs/provision/tasks.yaml
index 46f4faa..68716ce 100644
--- a/graphs/provision/tasks.yaml
+++ b/graphs/provision/tasks.yaml
@@ -105,4 +105,4 @@
   requires: [cobbler_disable_netboot]
   required_for: [set_status_provisioned]
   parameters:
-    timeout: 180
+    timeout: 900
